docs/root/start/sandboxes/gzip.rst

```diff
@@ -0,0 +1,128 @@
+.. _install_sandboxes_gzip:
+
+Gzip
+===========
```

Comment:
title underline should be the same length as the title

Reply:
Got.